src/core/best-practices.md

```diff
@@ -111,3 +111,47 @@ dispersion of 1 degree could classify two separate fixations instead of one. Set
 dispersion too high might group fixations together that should have been considered separate fixations. 
 Ensure the maximum dispersion accommodates your viewing task. 
 
+## Sychronization
+Pupil Core is often used concurrently with third-party devices and software (e.g. physiological sensors, 
+motion capture, stimuli presentation). In order to correlate the data between these, temporal alignment is of great importance.
+
+How you go about synchronizing Pupil Core with other devices or software will depend on the setup, but there are three 
+common approaches (steps 2–3 leverage our [Network API](/developer/core/network-api)):
+
+::: tip
+<v-icon large color="info">info_outline</v-icon>
+Read about Pupil Core's [timing definitions here](/core/terminology/#timing).
+:::
+
+::: tip
+<v-icon large color="info">info_outline</v-icon>
+When using multiple Pupil Core devices (running on the same or multiple machines), be sure to turn-on 
+the [Network Time Sync Plugin](https://docs.pupil-labs.com/core/software/pupil-capture/#pupil-time-sync) which takes 
+care of Pupil Time synchronization. The [Pupil Groups Plugin](/core/software/pupil-capture/#pupil-groups) also helps when 
+using more than one Pupil Core device simultaneously. 
+:::
+
+### 1. Lab Streaming Layer (LSL)
+We maintain a [Plugin for LSL](https://github.com/labstreaminglayer/App-PupilLabs/tree/master/pupil_capture) that 
+publishes gaze data in real-time using the [LSL framework](https://labstreaminglayer.readthedocs.io/info/intro.html). 
+This enables unified and time-synchronised collection of measurements with other 
+[LSL supported devices](https://labstreaminglayer.readthedocs.io/info/supported_devices.html).
+
+The plugin synchronizes Capture's own clock, Pupil Time, with the pylsl.local_clock(). This allows the recording of 
```

Comment:
@N-M-T consider internal link to terminology section where Pupil Time is defined.

Reply:
I placed an internal link to the timing terminology in a :::tip at line 121. This appears near the beginning of the synchronization best practices, so that the reader will know what timing is before progressing.
![sync](https://user-images.githubusercontent.com/25012222/122225305-d9e08980-ceac-11eb-8ec6-34877f349ee2.JPG)

Reply:
Great! Thanks for pointing this out. I'll close/resolve this then.